docs/admin/integrations/prometheus.md

```diff
@@ -189,18 +189,77 @@ deployment. They will always be available from the agent.
 
 <!-- End generated by 'make docs/admin/integrations/prometheus.md'. -->
 
+### Authorization metrics
```

Comment:
This is very nice, especially the examples.

It'll be slightly odd though to have one subsystem's metrics called out with an H3, while all the others are grouped together.

https://coder.com/docs/admin/integrations/prometheus#available-metrics

These metrics should rather show there, and perhaps the example queries can be moved to their own page.